src/pip/_internal/utils/outdated.py

```diff
@@ -30,6 +32,13 @@
 logger = logging.getLogger(__name__)
 
 
+def _get_statefile_path(cache_dir, key):
+    # type: (str, Union[str, Text]) -> str
+    key_bytes = ensure_binary(key)
+    name = hashlib.sha256(key_bytes).hexdigest()
+    return os.path.join(cache_dir, "{}-selfcheck.json".format(name))
```

Comment:
Might it be nicer to make the filename start with `selfcheck-` rather than end with it?

Reply:
Yes, much nicer. Fixed [here](https://github.com/pypa/pip/pull/6855/commits/ece6aacd9fb37ed94149053425f0c46d68a46055#diff-53b83fe57d94dba72021e5cbac099934R39).